modules/cert-manager-acme-dns01-explicit-gcp.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-explicit-gcp_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using explicit credentials for GCP Cloud DNS
+
+You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on GCP. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Google CloudDNS.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
```

Comment:
Not sure for which steps this prerequisite is. In this section, `oc edit certmanager cluster` needs it, `oc edit certmanager cluster` appears in all other dns-01 sections, but those sections do not list this prerequisite, this is inconsistent.

- If this prerequisite is is for `oc edit certmanager cluster`, then all other dns-01 sections need add it.

Reply:
But if this prerequisite is not extended for `oc edit certmanager cluster`, then let's look at remaining snippets:
oc create -f namespace.yaml
oc create -f issuer.yaml
oc create -f certificate.yaml

Only namespace creation `oc create -f namespace.yaml` needs cluster-admin, Issuer and Certificate creation need not. So, if this prerequisite is added for namespace creation only, I suggest replace it with `oc new-project my-issuer-namespace`, `oc new-project` needs not cluster-admin, as previous comment https://github.com/openshift/openshift-docs/pull/59702#discussion_r1233798844 .

Reply:
Added `oc new-project my-issuer-namespace` in all relevant places